apps/transport/lib/transport/stats_handler.ex

```diff
@@ -90,6 +90,74 @@ defmodule Transport.StatsHandler do
       nb_siri_lite: count_dataset_with_format("SIRI Lite"),
       count_geo_data_lines: count_geo_data_lines()
     }
+    |> Map.merge(gbfs_stats())
+  end
+
+  def gbfs_stats do
+    today = Date.utc_today()
+
+    # Latest metadata for each GBFS resource for today
+    rows =
+      DB.ResourceMetadata.base_query()
+      |> join(:inner, [metadata: m], r in DB.Resource, on: r.id == m.resource_id, as: :resource)
+      |> where([resource: r], r.format == "gbfs")
+      |> where([metadata: m], fragment("?::date", m.inserted_at) == ^today and fragment("? \\? 'stats'", m.metadata))
```

Comment:
https://elixirforum.com/t/ecto-fragments-and-the-postgresql-json-operator/15766/6